packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-expandable-answer.tsx

```diff
@@ -26,7 +26,7 @@ export class AtomicSmartSnippetExpandableAnswer {
   @Element() public host!: HTMLElement;
 
   @Prop({reflect: true}) expanded!: boolean;
-  @Prop({reflect: true}) htmlContent!: string;
+  @Prop() htmlContent!: string;
```

Comment:
Reflecting HTML or CSS is heavy on the DOM for no true reason since this content is inserted in the component.